src/odemis/gui/model/__init__.py

```diff
@@ -1385,4 +1385,4 @@ def __init__(self, name, **kwargs):
         self.interpolate_content.value = False
 
         self.mpp.value = 10e-6
-        self.mpp.range = (10e-6, 1e-3)
+        self.mpp.range = (1e-10, 1e10)  # (10e-6, 1e-3)
```

Comment:
Why making the maximum mpp so big? That's 10 million kilometers per pixel. We should raise an error if such value happens. Probably 1 is fine as maximum if you want really freedom.